modules/core/shared/src/test/scala/scaladex/core/model/ArtifactTests.scala

```diff
@@ -196,6 +196,20 @@ class ArtifactTests extends AnyFunSpec with Matchers:
         )
       )
     }
+
+    it("should allow creating Artifact with CompilerPlugin platform without exceptions") {
+      val artifact = createArtifact(
+        groupId = "org.example",
+        artifactId = "myplugin_2.13",
```

Comment:
This `artifactId` should not be treated as a compiler plugin. For a compiler plugin we expect the binary version to be a full Scala versions: `$major.$minor.$tag`.

For example: https://repo1.maven.org/maven2/org/typelevel/kind-projector_2.13.16/